client/app/shorts/[short]/page.tsx

```diff
@@ -10,61 +12,66 @@ import Image from "next/image";
 import { videos } from "@/lib/api/data";
 import MainTemplate from "@/components/templates/MainTemplate";
 import { redirect } from "next/navigation";
+import { useState } from "react";
 
 type ShortsComponentProps = {
   params: { short: string };
+  isReady: boolean;
```

Comment:
Remove unnecessary props for page component
![image](https://github.com/mart-anthony-stark/Youtub/assets/69393826/d79ba30e-c679-43c8-9bfb-eb46a119e576)